model/src/conditions/inline-condition-operators.js

```diff
@@ -92,3 +103,30 @@ function formatValue (fieldType, value) {
   }
   return `'${value}'`
```

Comment:
must've missed this in another PR, why are you returning string within a string?

Reply:
The resultant expr-eval expression needs to look something like "field = 'value'" - i.e. string values need to be wrapped in something identifying it as such.